src/viking.js

```diff
@@ -1,11 +1,174 @@
-// Soldier
-function Soldier() {}
+// // Soldier
+// function Soldier() {}
 
-// Viking
-function Viking() {}
+// // Viking
+// function Viking() {}
 
-// Saxon
-function Saxon() {}
+// // Saxon
+// function Saxon() {}
+
+// // War
+// function War() {}
+
+
+class Soldier {
```

Comment:
This is ES6 :)

Reply:
I don’t really know classes. The pre-work was the first time I ever used them. 

I just found the ES5 prototype syntax much more confusing. I started off using it but got stuck so I switched to using classes instead.

Are there cases where it would not be possible to use ES6 classes?